jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -83,17 +90,40 @@ def functionTest(String test_name, String label_name, String TEST_GROUP, Boolean
                     ){
                         try{
                             timeout(60){
+                                // Get main test scripts for un-manifest-src test
+                                if (this.TEST_TYPE != "manifest") {
+                                    def exists = fileExists 'RackHD'
+                                    if( !exists ){
+                                        echo "Checkout RackHD for un-src test."
```

Comment:
what is it ? a kind of unit-test ? or debug test ? if debug test , shouldn't put your private repo & branch in src code , I think.